.github/workflows/gh-pages.yml

```diff
@@ -10,16 +12,18 @@ jobs:
     steps:
       - name: SCM Checkout
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
 
       - name: Setup Python & Poetry Environment
-        uses: exasol/python-toolbox/.github/actions/python-environment@0.14.0
+        uses: exasol/python-toolbox/.github/actions/python-environment@0.18.0
 
       - name: Build Documentation
         run: |
-          poetry run python -m nox -s build-docs
+          poetry run nox -s docs:multiversion
```

Comment:
If you haven't used multi version before you likely miss a configuration bit, see here:
https://exasol.github.io/python-toolbox/0.20.0/faq.html#warning-while-building-multiversion-documentation

Reply:
not sure why it generated `docs:multiversion`. On subsequent `tbx workflow update` it was just `docs:build` (which worked fine)